src/main/java/org/teragrep/JavaPcre.java

```diff
@@ -132,10 +160,42 @@ public void pcre2_versioncheck(){
 //        compile_options = new LibJavaPcre.OptionsStruct();
 //    }
 
-    public void pcre2_compile_java(String pat, int size){
+    public void pcre2_gcontext_create(){
+        gcontext = LibJavaPcre.INSTANCE.pcre2_gcontext_create();
+    }
+    public void pcre2_ccontext_create(){
+        if (gcontext == null){
+            System.out.print("Error! general context is not initialized properly!\n");
```

Comment:
please change from System.out.println (sout) to slf4j-api use.

```
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-api</artifactId>
        <version>2.0.6</version>
      </dependency>

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

    private static final Logger LOGGER = LoggerFactory.getLogger(JavaPcre.class);
LOGGER.error("bogus input")
```

Reply:
Changed to using slf4j-api